ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -887,6 +877,13 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-external-lb-post
+- as: vsphere-ipi-multi-vcenter-f28
```

Comment:
hi @jinyunma Could you please let me know if wee need a upgrade job for `vsphere-ipi-multi-vcenter-f28` and `vsphere-upi-multi-vcenter-f28`?

Reply:
Let's add these 2 jobs first, if not need then we remove them.